src/dispatch/incident/service.py

```diff
@@ -81,6 +80,11 @@ def get_by_name(*, db_session, project_id: int, name: str) -> Optional[Incident]
     )
 
 
+def get_all_by_incident_type(*, db_session, incident_type_id: int) -> List[Optional[Incident]]:
```

Comment:
nit: s/incident/incidents

Reply:
I named this function to be consistent with the other functions in this file (e.g. `get_all_by_status()` and `get_all_last_x_hours_by_status()`).